example/install_rackhd.sh

```diff
@@ -0,0 +1,252 @@
+#!/bin/bash
```

Comment:
Please add a `set -e` just underneath the start, or `#!/bin/bash -e` at the top to force an immediate failure in the case that any script command returns a failure code internally.